package.json

```diff
@@ -77,5 +79,26 @@
       "path": "./webapp/dist/ddocs/medic/_attachments/css/inbox.css",
       "maxSize": "70 kB"
     }
-  ]
+  ],
+  "jest": {
+    "runner": "jest-runner-mocha",
+    "testMatch": [
+      "<rootDir>/sentinel/tests/**/*.js",
+      "<rootDir>/api/tests/mocha/**/*.js"
+    ],
+    "testEnvironment": "node",
+    "collectCoverageFrom": [
+      "<rootDir>/sentinel/tests/**/*.{js}"
+    ],
+    "coverageThreshold": {
+      "global": {
+        "branches": 100,
+        "functions": 100,
+        "lines": 100,
+        "statements": 100
```

Comment:
Does this really mean we expect 100% coverage?

Also, we should probably have a conversation about coverage as a team. I'm interested in hearing opinions about people's experiences and how useful they found it?